src/lib/github.js

```diff
@@ -58,6 +58,16 @@ export function getRepositories() {
   }
 }
 
+/**
+ * Return all indexed contents from localStorage as an array
+ */
+export function getIndexedContents() {
+  const indexedContents = localStorage.getItem(REPOS_CONTENTS_TREE_STORAGE_KEY);
+  return indexedContents === null || indexedContents === []
+    ? []
+    : indexedContents.split(",");
```

Comment:
I prefer if we store the encoding of the data in json. You can encode with `JSON.stringify(data)` and you can decode with `JSON.parse(data)`. The rest of the code in this file uses this convention as well.

Reply:
Yeah, I put a todo. How about this as a follow up PR? I'll have to change how the `refresh()` and the fuzzy search works.

Reply:
If you do that, this will poison the local storage, and you'll need special logic to handle both cases, which from my perspective is a lot more work, and unnecessary tech debt. What do you think?

Reply:
I actually have a slightly better idea -- how about we don't even bother storing the repo content index? Just make the remote request every time. And, only load the index for the current repo (i.e. they can only search file files in the current repo).

This has the advantage that the code is simpler, but also, caching of indexes can be handles by an upcoming feature I wanted to implement, where the cache is handled by a service worker. It essentially will stand between JS fetch commands and only update the result of the fetch command when necessary.

Reply:
Sure, my original reasoning for caching it is to reduce the number of `fetch` calls, since it has to do a recursive call for each directory. I don't think it's a huge deal, since requests are specific to each user/github token, so it doesn't affect the application as a whole, but this could be annoying if the repo gets more nested directory or grows bigger.

I'll refactor it to not cache, and just make a call every time the search bar renders.